coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -30,15 +30,26 @@ def __init__(self, documentation, language,
         :param range:         The position range of type TextRange.
         """
         self.documentation = documentation
-        self.language = language.lower()
-        self.docstyle = docstyle.lower()
+        self.docstyle_definition = docstyle_definition
         self.indent = indent
         self.marker = marker
         self.range = range
 
     def __str__(self):
         return self.documentation
 
+    @property
+    def language(self):
+        return self.docstyle_definition.language.lower()
+
+    @property
+    def docstyle(self):
+        return self.docstyle_definition.docstyle.lower()
+
+    @property
+    def metdata(self):
+        return self.docstyle_definition.metadata
```

Comment:
1. typo --> `metdata` --> `metadata`
2. you sure you want to expose this? someone could get the idea that `doc.parse()` is the same like `doc.metadata`^^